smf/internal-dns/config.toml

```diff
@@ -0,0 +1,22 @@
+[dropshot]
+bind_address = "[fd00:1234::4]:5353"
+request_body_max_bytes = 1048576
+
+[dns]
+bind_address = "[fd00:1234::4]:4753"
```

Comment:
I'd suggest we make this port 53 so that we can use standard DNS tools in their default configuration.

Reply:
Agreed for this context. I originally used 4753 in the example config to avoid conflicting with local DNS servers on dev machines, but that does not apply here.